packages/telnyx_webrtc/pubspec.yaml

```diff
@@ -18,7 +18,8 @@ dependencies:
   just_audio: ^0.9.42
   shared_preferences: ^2.2.3
   connectivity_plus: ^6.1.0
-
+  path_provider: ^2.1.5
```

Comment:
We need to review the pubspec for things that we really need. 

If we can somehow not use path_provider (which I don't think we need if we remove the file logger) or something that is super common like shared_preferences (might be a bit harder with out current implementation) it would be preferred

Ideally, for a package, we keep the dependencies as low as possible to avoid possible dependency conflicts with the client side implementation of our SDK.